apps/librelingo_json_export/tests/test_cli.py

```diff
@@ -101,10 +101,10 @@ def test_dry_run_calls_real_export_course(mocks, inputs, invoke):
 
 
 @pytest.mark.parametrize("no_dry_run_arg", [["--no-dry-run"], []])
-def test_creates_excpected_files(fs, invoke_with_no_mocks, no_dry_run_arg):
+def test_creates_excpected_files(f_s, invoke_with_no_mocks, no_dry_run_arg):
     input_path = Path(__file__).parent / "fixtures" / "fake_course"
-    fs.add_real_directory(input_path)
-    fs.add_real_directory(os.path.dirname(jsonschema.__file__))
+    f_s.add_real_directory(input_path)
```

Comment:
regarding this, I think it should be added to `"good-names"` like this: https://stackoverflow.com/a/37799332

my reason for that is because that is the name of a mock that I think wasn't defined in our repo, right? so I think we can allow that, especially if it's related to a convention